src/utils/telemetry/report-error.mjs

```diff
@@ -30,15 +31,17 @@ export const reportError = async function (error, config = {}) {
       user: {
         id: globalConfig.get('userId'),
       },
+      metadata: config.metadata,
       osName: process.platform,
```

Comment:
```suggestion
      osName: `${os.platform()} (${os.arch()})`,
```

from https://github.com/bugsnag/bugsnag-js/blob/f220383cb175cc779d5e6f7d015a8f71946f5a28/packages/plugin-node-device/device.js#L9

Reply:
Thoughts on adding a hyphen to this `${os.platform()}-(${os.arch()})`?